### ISSUE TITLE
quattor/client/rpms.pan: fix ncm-ncd

### ISSUE THREAD

**Issue by kwaegema** [quattor/template-library-core#84]
```diff
+    ncd = escape('ncm-ncd');
+    if(! exists(ncd)) {
+        SELF[ncd] = nlist();
```

**Comment by jrha** [comment]
So, why isn't this a PR?

**Comment by kwaegema** [comment]
it was merely a quick reminder to do that; done :)

**Comment by ned21** [comment]
Milestone removed to avoid blocking the release although it looks like #114 fixes this, in which case the milestone should be re-added

**Comment by kwaegema** [comment]
Indeed, #114 fixes #84
